wsdl-cli/src/main/java/io/ballerina/wsdl/cli/WsdlCmd.java

```diff
@@ -107,37 +107,41 @@ public WsdlCmd() {
 
     @Override
     public void execute() {
-        if (helpFlag) {
+        if (this.helpFlag) {
             StringBuilder stringBuilder = new StringBuilder();
             printLongDesc(stringBuilder);
-            outStream.println(stringBuilder);
+            this.outStream.println(stringBuilder);
+            exitOnError();
             return;
         }
-        Path modulePath = getModulePath();
-        if (!isValidBallerinaProject()) return;
-        if (!isValidModuleName(moduleName)) return;
-        if (inputPath.isEmpty()) {
-            outStream.println(MISSING_WSDL_PATH);
+        if (this.inputPath == null || this.inputPath.isEmpty()) {
+            this.outStream.println("A WSDL file path is required to generate the client and types.");
+            this.outStream.println("e.g: bal wsdl <wsdl file>");
             exitOnError();
             return;
         }
+        Path modulePath = getModulePath();
+        if (!isValidBallerinaProject()) return;
+        if (!isValidModuleName(this.moduleName)) return;
         try {
             if (Files.exists(modulePath) && !Files.isDirectory(modulePath)) {
-                outStream.printf(INVALID_DIRECTORY_PATH + "%n", moduleName);
+                this.outStream.printf(INVALID_DIRECTORY_PATH + "%n", this.moduleName);
                 exitOnError();
                 return;
             }
             if (Files.notExists(modulePath)) {
                 Files.createDirectories(modulePath);
             }
-            if (!Files.exists(Path.of(inputPath.get(0)))) {
-                outStream.println(inputPath.get(0) + " file does not exist.");
+            if (!Files.exists(Path.of(this.inputPath.get(0)))) {
```

Comment:
`this.inputPath.get(0)` can we extract this to a variable because this is accessed in multiple places